app/helpers/rad_common/application_helper.rb

```diff
@@ -80,12 +80,13 @@ def enum_to_translated_option(klass, enum, enum_value, default = enum_value.to_s
       I18n.t(key, default: default)
     end
 
-    def options_for_enum(klass, enum)
+    def options_for_enum(klass, enum, db_values=false)
```

Comment:
I feel like we should add a different method for db values rather than using the same method with a switch but I don't fully know what it's used for yet

Reply:
updated per our Slack conversation